config/application.rb

```diff
@@ -40,5 +40,8 @@ class Application < Rails::Application
     config.active_record.belongs_to_required_by_default = false
 
     ActiveSupport.to_time_preserves_timezone = false
+
+    # use structure.sql, not schema.rb
```

Comment:
Worth clarifying _why_ in this comment, e.g.

```suggestion
    # use structure.sql instead of schema.rb, as the latter does not support triggers. See:
    # https://edgeguides.rubyonrails.org/active_record_migrations.html#types-of-schema-dumps
```